modules/Emails/Email.php

```diff
@@ -2589,21 +2600,23 @@ public function handleBody($mail)
         global $current_user;
 
         // User preferences should takee precedence over everything else
-        $emailSettings = $current_user->getPreference('emailSettings',  'Emails');
+        $emailSettings = $current_user->getPreference('emailSettings', 'Emails');
         $alwaysSendEmailsInPlainText = $emailSettings['sendPlainText'] === '1';
 
         $sendEmailsInPlainText = false;
-        if(isset($_REQUEST['is_only_plain_text']) && $_REQUEST['is_only_plain_text'] === 'true') {
+        if (isset($_REQUEST['is_only_plain_text']) && $_REQUEST['is_only_plain_text'] === 'true') {
             $sendEmailsInPlainText = true;
         }
 
-        if($alwaysSendEmailsInPlainText === true) {
+        if ($alwaysSendEmailsInPlainText === true) {
             // plain text only
             $this->handleBodyInPlainTextFormat($mail);
-        } else if($alwaysSendEmailsInPlainText === false && $sendEmailsInPlainText === true) {
-            $this->handleBodyInPlainTextFormat($mail);
         } else {
-            $this->handleBodyInHTMLformat($mail);
+            if ($alwaysSendEmailsInPlainText === false && $sendEmailsInPlainText === true) {
```

Comment:
Same issue here if you use the code cleanup it will change 
`else if ($a = $ab){
//do something
} else {
//do something else
}
`
to

`else {
if ($a = $ab){
//do something
} else {
//do something else
}
}
`

as it is expecting 'elseif' and can't seem to correct it, so you have to manuall change any 'else if' to 'elseif' before you auto clean it